plugins/woocommerce/client/legacy/js/frontend/order-attribution.js

```diff
@@ -65,6 +65,8 @@
 		if ( ! allow ) {
 			// Reset cookies, and clear form data.
 			removeTrackingCookies();
+		} else if ( typeof sbjs === 'undefined' ) {
+			return; // Do nothing, as sourcebuster.js is not loaded.
```

Comment:
Why isn't the check the beginning of setOrderTracking(), like:

```
wc_order_attribution.setOrderTracking = function( allow ) {
 if ( typeof sbjs === 'undefined' ) {
    return;			
 }
 
 if ( ! allow ) {
			// Reset cookies, and clear form data.
			removeTrackingCookies();
  }
  ....
 }
 ```
 Do we still want to removeTrackingCookies(); if not allow, I presume?

Reply:
Yeah, in theory it would be to handle the case where a user accepted cookies, then went back and changed their preferences. The Sourcebuster library wouldn't be loaded, but still should remove any existing cookies.